home/models.py

```diff
@@ -584,3 +668,286 @@ class SimplePage(Page):
         edit_handlers.ObjectList(content_panels, heading='Content'),
         edit_handlers.ObjectList(general_panels, heading='General')
     ])
+
+
+class ManualsIndex(Page):
+    translated_title = TranslatedField('title')
+    title_en = models.CharField(
+        max_length=150, blank=True, verbose_name="Title")
+    title_de = models.CharField(
+        max_length=150, blank=True, verbose_name="Title")
+    title_it = models.CharField(
+        max_length=150, blank=True, verbose_name="Title")
+    title_fr = models.CharField(
+        max_length=150, blank=True, verbose_name="Title")
+    title_sv = models.CharField(
+        max_length=150, blank=True, verbose_name="Title")
+    title_sl = models.CharField(
+        max_length=150, blank=True, verbose_name="Title")
+    title_da = models.CharField(
+        max_length=150, blank=True, verbose_name="Title")
+
+    block_types = [
+        ('heading', core_blocks.CharBlock(classname="full title",
+                                          icon="title")),
+        ('paragraph', core_blocks.TextBlock(icon="pilcrow")),
+        ('rich_text', core_blocks.RichTextBlock(icon="pilcrow")),
+        ('info_block', InfoBlock()),
+        ('image', image_blocks.ImageChooserBlock(icon="image")),
+        ('wide_image', WideImageBlock(icon="image")),
+        ('images', InlineImagesBlock(icon="image")),
+        ('image_text_block_list', ImageTextBlockList()),
+    ]
+
+    body = TranslatedField('body')
+    body_en = StreamField(block_types, null=True,
+                          blank=True, verbose_name="body")
+    body_de = StreamField(block_types, null=True,
+                          blank=True, verbose_name="body")
+    body_it = StreamField(block_types, null=True,
+                          blank=True, verbose_name="body")
+    body_fr = StreamField(block_types, null=True,
+                          blank=True, verbose_name="body")
+    body_sv = StreamField(block_types, null=True,
+                          blank=True, verbose_name="body")
+    body_sl = StreamField(block_types, null=True,
+                          blank=True, verbose_name="body")
+    body_da = StreamField(block_types, null=True,
+                          blank=True, verbose_name="body")
+
+    subpage_types = [
+        'home.ManualsSectionPage',
+        'home.ManualsDetailPage'
```

Comment:
Why can I add a ManualsDetailPage here? I tried it, but the styling seems to be broken. Maybe instead we could add more available blocks to the Section Page, then users could add text also